docs/sdk/storage/fragments/ios/transfer-utility.md

```diff
@@ -354,84 +354,17 @@ transferUtility.uploadUsingMultiPart(data:data,
-
-```swift
-let getPreSignedURLRequest = AWSS3GetPreSignedURLRequest()
-getPreSignedURLRequest.bucket = "myBucket"
-getPreSignedURLRequest.key = "myFile.txt"
-getPreSignedURLRequest.httpMethod = .PUT
-getPreSignedURLRequest.expires = Date(timeIntervalSinceNow: 3600)  // Change the value of the expires time interval as required
-getPreSignedURLRequest.contentType = "text/plain"
-
-AWSS3PreSignedURLBuilder.default().getPreSignedURL(getPreSignedURLRequest).continueWith { (task:AWSTask<NSURL>) -> Any? in
-    if let error = task.error as? NSError {
-        print("Error: \(error)")
-        return nil
-    }
-
-    let presignedURL = task.result
-    // Use the Pre-Signed URL here as required
-    ....
-    ....
-
-    return nil
-}
-```
+When you upload objects to the S3 bucket the Amplify CLI creates, you must manually prepend the appropriate access-level information to the `key`. The correct prefix - `public/`, `protected/` or `private/` - will depend on the access level of the object as documented in the [Storage Access section](/sdk/storage/getting-started?platform=ios#storage-access).
 
 ## Transfer Utility and Pre-Signed URLS 
 
 The TransferUtility generates Amazon S3 pre-signed URLs to use for background data transfer. The best practice is to use Amazon Cognito for credentials with Transfer Utility. With Amazon Cognito Identity, you receive AWS temporary credentials that are valid for up to 60 minutes. The pre-signed URLs built using these credentials are bound by the same time limit, after which the URLs will expire. 
 
-Because of this limitation, when you use TransferUtility with AWS Cognito, the expiry on the Pre-Signed URLs generated internally is set to **50 minutes**. Transfers that run longer than the 50 minutes will fail.  If you are transferring a large enough file where this becomes a constraint, you should create static credentials using AWSStaticCredentialsProvider ( _see [Authentication](./authentication) for more details on how to do this_)  and increase the expiry time on the Pre-Signed URLs by increasing the value for the `timeoutIntervalForResource` in the Transfer Utility Options.  Note that the max allowed expiry value for a Pre-Signed URL is 7 days. 
+Because of this limitation, when you use TransferUtility with Amazon Cognito, the expiry on the Pre-Signed URLs generated internally is set to **50 minutes**. Transfers that run longer than the 50 minutes will fail.  If you are transferring a large enough file where this becomes a constraint, you should create static credentials using AWSStaticCredentialsProvider ( _see [Authentication](sdk/auth/how-it-works?platform=ios) for more details on how to do this_)  and increase the expiry time on the Pre-Signed URLs by increasing the value for the `timeoutIntervalForResource` in the Transfer Utility Options.  Note that the max allowed expiry value for a Pre-Signed URL is 7 days. 
```

Comment:
Link is in incorrect notation

Reply:
> Link is in incorrect notation

I was reviewing the PR preview link and seems like all the links work. Is there a reason to change them all?

Reply:
it is more for convention based on @harrysolovay 's guidance. I wouldn't block on this if the links work though.